src/spot_moveit_config/config/kinematics.yaml

```diff
@@ -1,14 +1,21 @@
-# Reference: https://moveit.github.io/moveit_tutorials/doc/trac_ik/trac_ik_tutorial.html
-arm:
+full_robot:
```

Comment:
May want to refer to: https://moveit.github.io/moveit_tutorials/doc/trac_ik/trac_ik_tutorial.html